src/main/java/club/sk1er/patcher/mixins/performance/WorldMixin_FixEarlyLightInitialization.java

```diff
@@ -0,0 +1,40 @@
+package club.sk1er.patcher.mixins.performance;
+
+//#if MC==10809
+import net.minecraft.util.BlockPos;
+import net.minecraft.world.EnumSkyBlock;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Constant;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.ModifyConstant;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(World.class)
```

Comment:
keep the imports and mixin annotation, mixin will scream at the mod if any mixin registered has no mixin annotation